README.md

```diff
@@ -31,3 +29,26 @@ export JIRA_AUTH=jira_username:jira_password
 
 /path/to/lts-backports-changelog.rb <VERSION> <PATH/TO/weekly.yml>
 ```
+
+# Usage in Docker
```

Comment:
Seems to make more sense as a new second-level entry directly below `generate-jenkins-changelog`.